### PR TITLE
Initialize GetDataValidation and GetConditionalFormat for sheets

### PR DESCRIPTION
**Pull request by chenliu1993**
Signed-off-by: chenliu1993 <13630583107@163.com>

# PR Details

## Description
This PR adds functions for getting data validations and conditional format strings across all sheets initially. 
This should fix initially for #827 

## Related Issue
#827 

## Motivation and Context
I am using this library to deal with excels during my work personally, it is convenient for me
And Just complete the function for these two fields

## How Has This Been Tested
Simply add one test for these two functions

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: datavalidation.go [hunk 1]
```diff
@@ -261,6 +261,29 @@ func (f *File) AddDataValidation(sheet string, dv *DataValidation) error {
...
+			return nil, err
+		}
+
+		// No data validaion on this sheet
+		if ws.DataValidations == nil {
+			return resultDVs, nil
+		}
+
+		for _, item := range ws.DataValidations.DataValidation {
+			dv := *item
```
Comment (xuri) [inline datavalidation.go:280]: The variable `item` is a pointer of the data validation, this code can be simplified by appending it to the result map directly.

File: styles.go [hunk 2]
```diff
@@ -2872,6 +2872,32 @@ func (f *File) SetConditionalFormat(sheet, area, formatSet string) error {
...
+		if err != nil {
+			return resultCF, err
+		}
+
+		resultCF[sheet] = []string{}
+		for _, cf := range ws.ConditionalFormatting {
+			cfRule := CfRule{SQRef: cf.SQRef}
+			for _, cr := range cf.CfRule {
+				cfRule.CfRule = *cr
+				cfRuleBytes, err := xml.Marshal(cfRule)
```
Comment (xuri) [inline styles.go:2890]: The result of this function should be same with the parameter `formatSet` of the `SetConditionalFormat` function. In another word, that the serilized string of `formatConditional`. So I think the new exported data type `CfRule` is not required.

**Review by xuri — CHANGES_REQUESTED** [review]
Thanks for your contribution. I've left some comments.

**Review by xuri — APPROVED** [review]

**Comment by chenliu1993** [comment]
Sorry for the late notice......

**Pull request merged**